src/build_multiversion.zig

```diff
@@ -79,14 +79,9 @@ pub fn main() !void {
 
     const target = try Target.parse(cli_args.target);
 
-    if (builtin.os.tag != .windows) {
-        // When we fetch llvm-objcopy in build.zig, there isn't an easy way to mark it as
-        // executable, so do it here.
-        const fd = try shell.cwd.openFile(cli_args.llvm_objcopy, .{ .mode = .read_write });
-        defer fd.close();
-
-        try fd.chmod(0o777);
-    }
+    // When we fetch llvm-objcopy in build.zig, there isn't an easy way to mark it as
+    // executable, so do it here.
+    try shell.file_make_executable(cli_args.llvm_objcopy);
```

Comment:
Could this be done with a system command `chmod +x` in `build_tigerbeetle_executable_get_objcopy`? I'm not sure if that's better... But it feels weird to have a build step that gets you the binary and you then need to make sure it's executable outside of that.

Reply:
I think the current solution is better than `chmod`, because system dependencies feel like a worse problem to me. But yeah, this _is_ pretty hacky.

The proper way to do that would be to write a custom `chmod`ing step, but that seems relatively large amount of code for little gain, given that the end-game is to not use vendored objcopy at all.

I'll write the custom step if you think thats worthwhile though!

Reply:
> I'll write the custom step if you think thats worthwhile though!

I think it's OK as-is for now! We do want to get rid of that step anyhow at some point. (and if someone else tries to use it in the meantime, figuring out the error will hopefully discourage them :smile:.)